### PR TITLE
Update Maven build and fix VERSION property

### LINKED ISSUE #108: jar file version and ST.VERSION mismatch
Not a serious issue but thought I would mention it.
I downloaded the ST jar file and download page said the version was 4.0.8 and the name of the jar file is ST-4.0.8.jar but the library reports itself as 4.0.7-SNAPSHOT using ST.VERSION.

### PR DESCRIPTION
**Pull request by sharwell**
- Restore full Maven build support
- Use a properties file to populate `ST.VERSION` accurately (fixes #108)

> :warning: To complete this, one of the following two things needs to happen:
> 1. The Ant build script is updated to copy the **version.properties** resource file and replace `${project.version}` with the appropriate value
> 2. The Ant build script is removed

:question: @parrt Does one of the two approaches listed here work for you? If so, I will go ahead and make the change and update this pull request.

### CODE REVIEW

**Comment by parrt** [comment]
is it easy to update ant script for (1)?

**Comment by sharwell** [comment]
> is it easy to update ant script for (1)?

It should be straightforward, especially if the only replacement property we need to support is `${project.version}`. I'll take a look soon.

**Comment by parrt** [comment]
Hi @sharwell this is a big change. Do we need this still?

**Comment by parrt** [comment]
Well, lemme just check to see if the silly thing I have now still works. If it does we can just leave it

File: pom.xml [hunk 1]
```diff
@@ -29,6 +30,71 @@
...
+			<roles>
+				<role>Developer - Maven stuff</role>
+			</roles>
+			<timezone>PST</timezone>
+		</developer>
+	</developers>
+
+	<licenses>
+		<license>
+			<name>BSD licence</name>
```
Comment (das-vinculum) [inline pom.xml:80]: I am sure it should be `BSD license`

**Comment by parrt** [comment]
This is ancient so I will close to be more tidy. going out with minor release shortly.

**Pull request closed without merging**